scripts/csv/discover_to_firebase.py

```diff
@@ -173,7 +200,7 @@ def _appendTADataFromRow(row, place):
     if totalReviewCount: ta_provider['totalReviewCount'] = int(totalReviewCount)
 
     ta_provider = {k: v for k, v in ta_provider.items() if v}  # rm empty keys.
-    if ta_provider: place['providers']['tripadvisor'] = ta_provider
+    if ta_provider: place['providers']['tripAdvisor'] = ta_provider
```

Comment:
Ugh, I'm not a fan of this mixed camel/lowercasing. At some point, I'd like for all these keys to be lowercase-only...